service-13/LargePacketTransferMessageParts.asn1

```diff
@@ -0,0 +1,42 @@
+--!
+--! Copyright (C) 2018 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Library was developed under a programme and funded by
+--! European Space Agency.
+--!
+--! This Library is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU General Public License as published by
+--! the Free Software Foundation, either version 3 of the License, or
+--! (at your option) any later version.
+--!
+--! This Library is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+LargePacketTransferMessageParts DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    PUSC-UINT32 FROM BasicTypes;
+
+uplinkMaximumPartSize PUSC-UINT32 ::= 1024
+downlinkMaximumPartSize PUSC-UINT32 ::= 1024
+
+UplinkMessagePart ::= MessagePartGeneric {PUSC-UINT32, uplinkMaximumPartSize}
+DownlinkMessagePart ::= MessagePartGeneric {PUSC-UINT32, downlinkMaximumPartSize}
+
+MessagePartGeneric {UnsignedInteger-Type, PUSC-UINT32:maximumPartSize} ::= SEQUENCE
```

Comment:
I prefer parametrization "by purpose" not "by type"

for example - are `partSequenceNumer` and `largeMessageTransactionIndentifier` required by standard to be of same size? IMHO `MessageTransactionId-Type` and  `PartSequenceNumber-Type` are better parameters for this type